simulators/ethereum/sync/sync.go

```diff
@@ -124,20 +133,72 @@ func (n *node) checkSync(t *hivesim.T, wantNumber uint64, wantHash common.Hash)
 	}
 }
 
-// head returns the node's chain head.
-func (n *node) head() (*types.Header, error) {
-	ctx, _ := context.WithTimeout(context.Background(), 5*time.Second)
-	return ethclient.NewClient(n.RPC()).HeaderByNumber(ctx, nil)
+type rpcRequest struct {
+	Method string
+	Params []json.RawMessage
+}
+
+func (n *node) triggerSync(t *hivesim.T) error {
+	// Load the engine requests generated by hivechain.
+	var newpayload, fcu rpcRequest
+	if err := common.LoadJSON("chain/headnewpayload.json", &newpayload); err != nil {
+		return err
+	}
+	if err := common.LoadJSON("chain/headfcu.json", &fcu); err != nil {
+		return err
+	}
+
+	// engine client setup
+	token := [32]byte{0x73, 0x65, 0x63, 0x72, 0x65, 0x74, 0x73, 0x65, 0x63, 0x72, 0x65, 0x74, 0x73, 0x65, 0x63, 0x72, 0x65, 0x74, 0x73, 0x65, 0x63, 0x72, 0x65, 0x74, 0x73, 0x65, 0x63, 0x72, 0x65, 0x74, 0x73, 0x65}
```

Comment:
is this a hive default somewhere?

Reply:
Yes. We hard-code this token in the client definition launcher scripts.